### DISCUSSION TITLE
How do I prevent this dialog box from showing up on startup?

### DISCUSSION THREAD

**Discussion by rp1231** [QL-Win/QuickLook#1350] [Q&A] [answered]
<img width="307" alt="QuickLook_fR0ILWIGRD" src="https://github.com/QL-Win/QuickLook/assets/47380919/04864945-87a8-46bd-9167-aa0ecd39a4d3">

**Comment by rabelux** [comment, marked as answer]
Have a look at #1281, #1063 and #577

**Comment by rp1231** [comment]
Thanks, I didn't have two installations but I was using a program called elevated startup and that had a shortcut of quicklook enabled at startup.
